bugwarrior/command.py

```diff
@@ -15,6 +16,7 @@
 )
 
 import logging
+logging.basicConfig()
```

Comment:
Without this I'm seeing "no handler available for logger". This is not needed as a result of this change, just none of the log calls were showing anything for some reason. This may be a regression from #331 or may be a result of my set up.